hack/machineset.sh

```diff
@@ -185,7 +185,7 @@ get_vsphere_ms() {
 
   # set golden image template name
   # TODO: read from parameter
-  template="windows-golden-images/windows-server-2022-template"
+  template="windows-golden-images/windows-server-2022-template-containersDisabled"
```

Comment:
this commit should be after the functionality commit

Reply:
addressed this and rest of the minor comments.